Mathlib/Data/ENNReal/Basic.lean

```diff
@@ -667,12 +667,12 @@ theorem le_of_forall_pos_le_add (h : ∀ ε : ℝ≥0, 0 < ε → b < ∞ → a
   exact ⟨r, hr0, h.trans_le le_top, hr⟩
 #align ennreal.le_of_forall_pos_le_add ENNReal.le_of_forall_pos_le_add
 
-theorem coe_nat_lt_coe {n : ℕ} : (n : ℝ≥0∞) < r ↔ ↑n < r :=
-  ENNReal.coe_nat n ▸ coe_lt_coe
-#align ennreal.coe_nat_lt_coe ENNReal.coe_nat_lt_coe
+theorem natCast_lt_coe {n : ℕ} : (n : ℝ≥0∞) < r ↔ ↑n < r :=
```

Comment:
What am I missing about this statement that makes it not a tautology? The current name is probably wrong, assuming there's an extra coercion here.

Reply:
No, the name is actually correct. The statement is `Nat.cast n < ENNReal.ofNNReal r ↔ Nat.cast n < r`

Reply:
I've rewritten it to `n < (r : ℝ≥0∞) ↔ n < r`. Hopefully that's clearer.